net/datacenter-node-install/setup-dc-node-1.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+if [[ "$EUID" -ne 0 ]]; then
```

Comment:
Something I noticed while doing the cleanup round...

Depending on how the user takes root, some of the variables we make assumptions about in this script may have unexpected values.  Affected are `USERNAME` (Both the normal shell global and the result of `whoami` used here) and `HOME`.

Reply:
The most obvious solution I see here is to enforce that the scripts are called with root taken via `sudo` rather than from a root shell (ala. `sudo su --`).  Then we'll have the `SUDO_*` envvars to detect the enforcement *and* fill out all of our script-local variables.

How does that sound to you @danpaul000 ?

Reply:
Yeah, I think it's reasonable to ask the user to just call `sudo this_script.sh`, as that is what the instruction is in the README, so now a future user has no excuse. :)

Reply:
I'll put the checks in place and make the appropriate changes tomorrow

Reply:
Addressed in https://github.com/solana-labs/solana/pull/5517/commits/85b7b72cfc0b597e13b4d533ab993e5d8d3ab862